frontend/cypress/integration/featureFiles/service_details.feature

```diff
@@ -70,17 +70,3 @@ Feature: Kiali Service Details page
     When user sees a minigraph
     And user chooses the "Show node graph" option
     Then the graph type is disabled
-
```

Comment:
Are these scenarios covered in other tests?

Reply:
This is an automation to this older bug: https://github.com/kiali/kiali/issues/4797. 
It can be duplicated for Multicluster as well, but I am not sure if it is really necessary.